angular.js

```diff
@@ -4,5 +4,33 @@ module.exports = {
         'plugin:@angular-eslint/recommended--extra',
```

Comment:
All of the `recommended--extra` rules are [variations to rules at a lower level of definition](https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended--extra.json) (js rules, typescript rules). I think we should not extend from this file to avoid inconsistency with the lower-level rulesets.

Reply:
What is the issue with doing so? We currently override all of these, but that doesn't mean there won't be Angular specific lower level configurations in the future. The point of recommended--extra is to take the lower level configurations into account with respect to Angular which I think is reasonable just the same as it was for TypeScript.

Reply:
> What is the issue with doing so? We currently override all of these, but that doesn't mean there won't be Angular specific lower level configurations in the future

If new ones are introduced my assumption is that we  would want to evaluate them and align with the lower-level rules. Therefore we would always be overriding all the rules. What value does it add? It seems like including it only adds risk that the lower level rules will diverge with the higher-level rules.

Reply:
Given that the purpose of this is to evaluate lower level rules with respect to Angular, I don't think divergence is a risk. We should be evaluating this rule set, and if there is divergence, it will be explicit and due to Angular. I'm coming at it from the perspective that extending it forces us to evaluate it and, although we agreed that the existing configuration should go into our lower level set, that may not always be the case. Maybe it would be helpful to extend the tool set to include a command that identifies that intersection between configurations?

Reply:
> Maybe it would be helpful to extend the tool set to include a command that identifies that intersection between configurations?

I think that would be useful tool, good idea! I expect that is not something we will tackle in this PR. Could you add a work item to the Web Chapter backlog?

In the interim can we add a section to the CONTRIBUTING.md that describes that during upgrades one should evaluate the `plugin:@angular-eslint/recommended--extra` rules to make sure they do not diverge from other rulesets? I'm still concerned that future authors can easily overlook these rules and that these rules in particular are more likely to diverge from existing rules than other rules we have extended.

Reply:
I'll go ahead and approve this to not block the follow-up PR discussed offline. If we can add the docs or tool in the follow-up that would be great.

Reply:
I've created work item [1501560](https://dev.azure.com/ni/DevCentral/_backlogs/backlog/NI%20Web%20Chapter/Work%20Items/?workitem=1501560).